sockeye/translate.py

```diff
@@ -72,12 +72,20 @@ def run_translate(args: argparse.Namespace):
         use_cpu = True
     device = pt.device('cpu') if use_cpu else pt.device('cuda', args.device_id)
     logger.info(f"Translate Device: {device}")
+    knn_index = args.knn_index
```

Comment:
why not pass the args members directly?

Reply:
I'm not sure if I should change the existing signature of a high-level function such as `run_translate`, but if you think this is acceptable then of course we could pass `knn_index` directly.